apps/slackbot/bolt_app.py

```diff
@@ -4,56 +4,75 @@
 ##############################################
 
 import os
+from os import environ
 from dotenv import load_dotenv
 from flask import Flask, request
-
 from scrape.prompt_reconstructor import PromptReconstructor
-load_dotenv()
 from langchain.chat_models import ChatOpenAI
-from os import environ
 from vectorstores import ConversationStore, LocalChromaStore
 from slack_bolt import App
 from slack_bolt.adapter.flask import SlackRequestHandler
 from tools import get_tools
 from task_agent import TaskAgent
 
 
-
-# This `app` represents your existing Flask app
-app = App(
-    token=os.environ.get("SLACK_OAUTH_TOKEN"),
-    signing_secret=os.environ.get("SLACK_SIGNING_SECRET"),
-)
-
-
 #####################################################################################################
-# Setting up environment variables and Slack configuration:
-# The code retrieves various environment variables using os.environ.get() method.
-# Environment variables include Slack signing secret, OAuth token, verification token, and OpenAI key.
+# Load environment variables and instantiate Slack client:
 #####################################################################################################
+load_dotenv()
```

Comment:
I think this step has to be before line 13 because some of the modules (e.g. `vectorstores.py`) will make use of the environment variables

Reply:
@20001LastOrder Ah, I see, there are more files which access environment variables that I missed earlier. I pushed some more commits that wrap up all the environment variable access in a new module `config.py`. Please review when you have time.